DllImportGenerator/DllImportGenerator.IntegrationTests/CharacterTests.cs

```diff
@@ -7,12 +7,6 @@ namespace DllImportGenerator.IntegrationTests
 {
     partial class NativeExportsNE
     {
-        [GeneratedDllImport(nameof(NativeExportsNE), EntryPoint = "unicode_return_as_uint")]
-        public static partial uint ReturnDefaultAsUInt(char input);
```

Comment:
Do we not want these tests?

Reply:
We are now requiring explicit marshalling information for `char` and `string`. I added these to the failure cases in the unit tests.